include/aws/common/hash_table.h

```diff
@@ -321,6 +322,10 @@ int aws_hash_table_remove_element(struct aws_hash_table *map, struct aws_hash_el
  *     element (if not set, iteration stops)
  * # AWS_COMMON_HASH_TABLE_ITER_DELETE   - Deletes the current value and
  *     continues iteration.  destroy_fn will NOT be invoked.
+ * # AWS_COMMON_HASH_TABLE_ITER_ERROR   - Error out and stop iteration not
```

Comment:
This is very hard to understand grammar wise.

Reply:
Sorry about it... Updated the comment to be more grammar-correct.